media/css/products/vpn/components/affiliate.scss

```diff
@@ -33,6 +33,7 @@ $image-path: '/media/protocol/img';
     }
 
     .vpn-affiliate-notification-reject {
+        display: block;
```

Comment:
Small bug fix to make sure the button is always on a new line when longer strings wrap:

![image](https://user-images.githubusercontent.com/400117/161546114-37978910-e6ad-48f8-85b8-8cf36dfeec70.png)